File: StudiumAnalytics-stable/spark/apps/studium-analytics-streaming-1.0.0/src/main/java/studium/analytics/streaming/Models/Item.java
package studium.analytics.streaming.Models;


import java.io.Serializable;

import com.google.gson.*;
import java.sql.Timestamp;

public class Item implements Serializable {
    private static final long serialVersionUID = 1L;
    public Teaching teaching;
    public String date;
    public String title;    
    public String toString()
    {
        Gson serializer = new Gson();
        String itemJson = serializer.toJson(this);
        String s1 = "{\"timestamp\" : "+new Timestamp(System.currentTimeMillis()).getTime()+",";
        s1 += itemJson.substring(1, itemJson.length());
        return s1;
    }
}

File: StudiumAnalytics-stable/dotnetBackend/StudiumAnalytics/StudiumAnalyticsBackend/Models/Item.cs
using System;
using System.Text.Json;

namespace StudiumAnalyticsBackend.Models
{
    public class Item
    {
        public Teaching teaching { get; set; }

        public string date { get; set; }

        public string title { get; set; }

        public override string ToString()
        {
            return JsonSerializer.Serialize<Item>(this);
        }
    }
}
